### ISSUE TITLE
How do I get the dimensions (height and width) of pictures

### ISSUE THREAD

**Issue by mrboring** [mono/taglib-sharp#329]
Hi

I have mp3 files with pictures. I want to check that the height and width are not greater than 300 pixels. I can see `Tag.Pictures` which contains various information, but not the dimensions. I'm guessing that the `Data` property might contain this information.

How do I get the dimesions?

Thanks.

**Comment by jerry08** [comment]
@mrboring I don't think you can get those details from [taglib-sharp](https://github.com/mono/taglib-sharp). I suggest you use [ImageSharp](https://github.com/SixLabors/ImageSharp) to get the image info. For example:

```c#
using SixLabors.ImageSharp;
using TagFile = TagLib.File;
```

```c#
var filePath = "...";
var file = TagFile.Create(filePath);
var pics = file.Tag.Pictures;

var imgBytes = pics[0].Data.Data;

var image = Image.Load(imgBytes);
var height = image.Height;
var width = image.Width;
```

**Comment by mrboring** [comment]
@jerry08, Thanks, this is very helpful.
